### PR TITLE
DMS: Improvements from testing

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

I tested the workflow and want to add a few tweaks to make some sections more clear. 
I also added some missing information to provide more clarity.

### CODE REVIEW

File: advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx [hunk 1]
```diff
@@ -85,64 +89,70 @@ Consider the following ID guidelines:
...
+
+Created from the Certificate Authority configured in `TLS_CA_PATH`.
+
+The Apicurio client uses it to perform mTLS authentication with the `transporter-rw-service`.
+
+### `DBZ_DATABASES`
+
+This is a list of source database information you require for the EDB DMS Reader be able to read the correct source database information for the migration. 
+
+You can configure the EDB DMS Reader to migrate multiple databases. The `DBZ_DATABASES_0__TYPE` section delimits the information for the first database. You can use `DBZ_DATABASES_1__TYPE` to provide data for a second database. Add more sections to the EDB DMS Reader (`DBZ_DATABASES_2__TYPE`, `DBZ_DATABASES_3__TYPE`) by increasing the index manully.
```
Comment (ksnavely) [inline advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx:131]: I didn't notice before, but I'm not sure why we use double underscores for these. @jperozoa I couldn't figure it out at a glance, do you know?

Comment (gvasquezvargas) [reply advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx:131]: maybe @zahid75 knows?

Comment (zahid75) [reply advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx:131]: I didn't try (as a matter of fact I was not aware of it) it earlier, will confirm now

Comment (zahid75) [reply advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx:131]: I confirmed, one can use the same reader instance to point to multiple source databases (by using _DATABASES_N__ pattern), and each database will appear as a separate source e.g. I defined two source PG sources and each appeared as a separate source on Sources page e.g.  "ora2pg (Postgres) zi-p2e-aug29-2" and "soe (Postgres) zi-p2e-aug29-2".

**Pull request merged**